install_config/syncing_groups_with_ldap.adoc

```diff
@@ -969,14 +964,13 @@ fields. For example, identify the users of a Group by their e-mail,
 and use the name of the Group as the common name. The following configuration
 file creates these relationships.
 
-.LDAP Sync Configuration Using Augmented Active Directory Schema With Recursive Members: *_augmented_active_directory_config_recursive.yaml_*
+.LDAP Sync Configuration Using Augmented Active Directory Schema With Nested Members: *_augmented_active_directory_config_nested.yaml_*
 ====
 [source,yaml]
 ----
 kind: LDAPSyncConfig
 apiVersion: v1
 url: ldap://LDAP_SERVICE_IP:389
-insecure: true
```

Comment:
Because it is not relevant to the example.  I only left the others since they had callouts to them.  Ideally they would reference the CA field, but I would prefer to limit this already overly large diff.